### PR TITLE
[IMP] generic : All publishable record are set by default to website_…

### PR DESCRIPTION
**Pull request by tivisse**
…published=False
TASK : https://github.com/odoo/odoo/compare/master...odoo-dev:master-unpublished-default-yti?expand=1
PAD : https://pad.odoo.com/p/r.6g3Z8IkioYVqssJl

### CODE REVIEW

File: addons/delivery/delivery_view.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
             <field name="model">delivery.carrier</field>
             <field name="arch" type="xml">
                 <form string="Carrier">
+                    <sheet>
```
Comment (tde-banana-odoo) [inline addons/delivery/delivery_view.xml]: Why this change ?

File: addons/website_partner/models/res_partner.py [hunk 2]
```diff
@@ -42,5 +42,6 @@ def _website_url(self, cr, uid, ids, field_name, arg, context=None):
         return res
 
     _defaults = {
-        'website_published': True
+        'website_published': False,
+        'website_private': True,
```
Comment (tde-banana-odoo) [inline addons/website_partner/models/res_partner.py]: Just setting website_private should be sufficient

File: addons/delivery/delivery_view.xml (hunk 1 above)
Comment (tivisse) [inline addons/delivery/delivery_view.xml]: More pretty + in accordance with most of the form views.

Comment (tde-banana-odoo) [inline addons/delivery/delivery_view.xml]: Do it in a specific commit then.

**Comment by tivisse** [comment]
Landed in master at https://github.com/odoo/odoo/commit/bf8bc292d6cff4f10a8032cdee3e1ed20b2c6992
